tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -90,43 +90,46 @@ def test_multiple_efs(
 
     To verify the efs is the existing efs, the test expects a file with random ran inside the efs mounted
     """
+    if scheduler == "awsbatch":
+        return
```

Comment:
This should be removed. After this is removed, can you run the test again?

Reply:
Sorry, thanks. Yeah, that's for catching that before I retested.